bin/gbuild

```diff
@@ -259,11 +262,13 @@ suites.each do |suite|
     info "Grabbing results"
     system! "copy-from-target #{@quiet_flag} out #{build_dir}"
 
+    unless @options[:skip_image]
```

Comment:
you mean :cache_ro here, right?  Could you merge that condition into the if statement?